vendor/sezo/sezoel.yaml

```diff
@@ -2,20 +2,13 @@ name: SEZO EL
 description: Environment Sensor
 
 # Hardware versions (optional, use when you have revisions)
-hardwareVersions:
-  - version: '1.0'
-    numeric: 1
-  - version: '1.0-rev-A'
-    numeric: 2
 
 # Firmware versions (at least one is mandatory)
 firmwareVersions:
   - # Firmware version
-    version: '2.0'
```

Comment:
If it is a new firmware version do not delete the older one, this can make a bad behavior on the devices already working with it, please add it below.